07-routing-gossip.md

```diff
@@ -967,6 +976,8 @@ A node:
   - SHOULD monitor the funding transactions in the blockchain, to identify
   channels that are being closed.
   - if the funding output of a channel is being spent:
+    - if a `channel_update` had `splicing` in the `channel_flags`:
+      - SHOULD delay 10 blocks considering the output spent.
```

Comment:
If we're delaying all funding spends by 12 blocks, this shouldn't be necessary anymore